src/index.js

```diff
@@ -1,48 +1,107 @@
 import './style.css';
 
-// Get the necessary HTML elements
+const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+let gameId = 'Zl4d7IVkemOTTVg2fUdz';
+
+const createGame = async (gameName) => {
+  const response = await fetch(`${API_URL}/games`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: gameName }),
+  });
+
+  const data = await response.json();
+
+  gameId = data.result;
+};
+
+const getScores = async () => {
+  const response = await fetch(`${API_URL}/games/${gameId}/scores`);
+  const data = await response.json();
+
+  if (!data || !data.result) {
+    throw new Error('Invalid response from server');
+  }
+
+  return data.result;
+};
+
```

Comment:
- [ ] Great job implementing the reqeust from previous reviewe.  However, there're some issues you still need to work o before your project meet all the requirements for the milestone.
-  Second, When the Refresh button is click, it should hit the GET API endpoint causing the scores list to change. But your app behaves differently to this. Kindly look into it and fix.


### Your app behaviou
[app-behave.webm](https://user-images.githubusercontent.com/26597432/225344894-0759e0d5-154a-4f86-9d1f-489889328c88.webm)

Reply:
@Qoosim when the refresh button is clicked , should give all the scores in the api or should I set some limit